19/transaction_network_election_v2.0.yaml

```diff
@@ -0,0 +1,53 @@
+---
+"$schema": "http://json-schema.org/draft-04/schema#"
+type: object
+title: Network Election Schema - Propose a network change
+required:
+- operation
+- version
+- asset
+- outputs
+properties:
+  operation: "VALIDATOR_ELECTION"
+  # version should greater than or equal the base transaction spec version
+  # - major number should be equal to the base transaction spec major version
+  # - minor number should be greater than or equal to base transaction spec minor version
+  version:
```

Comment:
I think, multiple versions make sense only when there are moving parts. Since "VALIDATOR_ELECTION" and "CREATE" transactions will always be processed by the same soft, it makes sense to use a single version for them.

Reply:
I concur, as of now, the version is same as the `CREATE` transaction spec. The only reason we might need a minor version greater than the `CREATE` spec version number is when the `asset.data` spec is changed for whatever reason. I don't see that happening anywhere in the future but I have left room for such possibility.